onnx/backend/test/case/node/attributehasvalue.py

```diff
@@ -0,0 +1,48 @@
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np  # type: ignore
+
+import onnx
+
+from ..base import Base
+from . import expect
+
+
+class AttributeHasValue(Base):
+    @staticmethod
+    def export() -> None:
+        def test_one_attribute(name, **kwargs):
+            node = onnx.helper.make_node(
+                "AttributeHasValue",
+                inputs=[],
+                outputs=["output"],
+            )
+
+            output = np.array(False)
+            expect(
+                node,
+                inputs=[],
+                outputs=[output],
+                name="test_attribute_has_value_{name}_false".format(name=name),
+            )
+
+            node = onnx.helper.make_node(
+                "AttributeHasValue",
+                inputs=[],
+                outputs=["output"],
+                **kwargs,
+            )
+
+            output = np.array(True)
```

Comment:
May be replace `True` by `bool(kwargs)` ... then you can use the same test-case with no-attributes as well as one-or-more attributes.